src/main/java/mate/academy/ConnectionUtil.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/jv_jdbc_intro";
+    private static final Properties BD_Properties;
```

Comment:
There is a typo in the variable name `BD_Properties`. It should be `DB_Properties` to reflect that it holds database properties.